artemis-server/src/main/java/org/apache/activemq/artemis/core/config/impl/Validators.java

```diff
@@ -122,6 +122,30 @@ public void validate(final String name, final Object value) {
       }
    };
 
+   public static final Validator POSITIVE_INT = new Validator() {
```

Comment:
This isnt needed, simply change the xsd to int and it would fail

Reply:
ignore i spotted, why finally, its because GB etc can be used in the string type, and then converted as such cannot limit this in xsd.